velox/exec/tests/HashPartitionFunctionTest.cpp

```diff
@@ -114,3 +114,17 @@ TEST_F(HashPartitionFunctionTest, HashPartitionFunction) {
     EXPECT_EQ(4, functionWithBits2.numPartitions());
   }
 }
+
+TEST_F(HashPartitionFunctionTest, HashPartitionFunctionSpec) {
```

Comment:
naming: HashPartitionFunctionSpec -> spec

Can we add test for RoundRobinXxxSpec as well?